### PR TITLE
[Athlete Page] - Fix Price Overview Section

### PR DESCRIPTION
**Pull request by jayesh2408**
# Description
Fixed athlete page according to the ticket which was fixing to MP:BV value text , hiding some text etc.

Fixes Jira Ticket # 540
https://athletex.atlassian.net/browse/AX-540

## Type of change
Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Is this a UI Change? If so please include screenshot of before and after states below
## Before
image

## After
image

# How Has This Been Tested?
This application was tested on Chrome (flutter run -d web-server)

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have assigned 2 reviewers to check my work
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: lib/pages/AthletePage.dart [hunk 1]
```diff
@@ -1436,7 +1441,7 @@ class _AthletePageState extends State<AthletePage> {
                                                       12, false, false))),
                                         ]),
                                         Text(shortBookValue,
-                                            style: textStyle(greyTextColor, 14,
+                                            style: textStyle(greyTextColor.withOpacity(0), 14,
```
Comment (Friendlygeckos) [inline lib/pages/AthletePage.dart:1444]: why do we have `.withOpacity(0)`? I do not see a difference in the text

File: lib/pages/AthletePage.dart [hunk 2]
```diff
@@ -741,14 +741,17 @@ class _AthletePageState extends State<AthletePage> {
                                                     MainAxisAlignment
                                                         .spaceBetween,
                                                 children: <Widget>[
+
```
Comment (Friendlygeckos) [inline lib/pages/AthletePage.dart:744]: I do not think it is necessary to have the extra spacing here

File: lib/pages/AthletePage.dart [hunk 3]
```diff
@@ -757,12 +760,14 @@ class _AthletePageState extends State<AthletePage> {
                                                                 false,
                                                                 false))),
                                                   ]),
-                                                  Text(bookValue,
+                                                  Visibility(
```
Comment (Friendlygeckos) [inline lib/pages/AthletePage.dart:763]: Why are we using a visibility widget if we are not going to toggle the visibility at all?

File: lib/pages/AthletePage.dart [hunk 4]
```diff
@@ -757,12 +760,14 @@ class _AthletePageState extends State<AthletePage> {
...
+                                                  visible: false,
+                                                  child:Text(bookValue,
                                                       style: textStyle(
-                                                          greyTextColor,
+                                                          greyTextColor.withOpacity(1),
```
Comment (Friendlygeckos) [inline lib/pages/AthletePage.dart:767]: Why do we have `.withOpacity(1)`? I do not see a difference in the text

**Review by Friendlygeckos — CHANGES_REQUESTED** [review]
- You still need to fix the MP:BV ratios on the athlete page. Right now they are still showing the same values for all the athletes.
- You also still need to remove the all time high book value and market price. Right now they are currently present on the athlete page

File: lib/pages/AthletePage.dart [hunk 5]
```diff
@@ -741,14 +741,17 @@ class _AthletePageState extends State<AthletePage> {
...
                                                 children: <Widget>[
+
                                                   Row(children: <Widget>[
                                                     Text(bookValue,
                                                         style: textStyle(
                                                             Colors.white,
                                                             12,
                                                             false,
                                                             false)),
+
```
Comment (Friendlygeckos) [inline lib/pages/AthletePage.dart:752]: Remove the space

File: lib/pages/AthletePage.dart [hunk 6]
```diff
@@ -741,14 +741,17 @@ class _AthletePageState extends State<AthletePage> {
...
                                                   Row(children: <Widget>[
                                                     Text(bookValue,
                                                         style: textStyle(
                                                             Colors.white,
                                                             12,
                                                             false,
                                                             false)),
+                                                 
                                                     Container(
+
```
Comment (Friendlygeckos) [inline lib/pages/AthletePage.dart:754]: Remove the space here aswell

**Review by Friendlygeckos — CHANGES_REQUESTED** [review]

**Review by user2745 — APPROVED** [review]

**Review by mretana1999 — APPROVED** [review]

**Review by Friendlygeckos — APPROVED** [review]

**Pull request merged**